core.rb

```diff
@@ -9,6 +9,7 @@
 require 'aws-sdk'
 require 'stomp'
 require 'open-uri'
+require 'nokogiri'
```

Comment:
I guess this introduces a (pretty heavy) dependency? and so should be avoided if possible; if the comment below about checking the package name from the DESCRIPTION file is appropriate, then this dependency could be removed

Reply:
if there is another way to read in the file and parse it I'm open to suggestions.   The database still needs the correct link

Reply:
The organization capitalization also has to be correct  (ie.  Bioconductor/spbtest3  couldn't be   bioconductor/sbptest3)   and the organization of hte github account isnt in the description

Reply:
So the description file can't be used

Reply:
I didn't realize that the 'organization' (or sometimes individual, i guess) part of the URL needed to match; would it be possible to be more explicit in the error message, e.g., the "Capitalization in the url provided above {{}} must match capitalization of the repository url {{}}" or if substitution is difficult then be explicit about what needs to match with a text description?

Reply:
I'll look more into seeing if there is a better dependency that could parse the html page -  or maybe at least a lighter one - 

Yes that should be doable the more specific message should be doable - I'll just have to change the helper functions a little bit and possible add a different one...

Reply:
nokogiri is what mostly comes up in searches and is the top recommended on the ruby-toolbox site ... https://www.ruby-toolbox.com/categories/html_parsing.html